src/tools/tidy/src/ui_tests.rs

```diff
@@ -12,11 +12,11 @@ use std::path::{Path, PathBuf};
 // should all be 1000 or lower. Limits significantly smaller than 1000 are also
 // desirable, because large numbers of files are unwieldy in general. See issue
 // #73494.
-const ENTRY_LIMIT: usize = 900;
+const ENTRY_LIMIT: u32 = 900;
 // FIXME: The following limits should be reduced eventually.
 
-const ISSUES_ENTRY_LIMIT: usize = 1676;
-const ROOT_ENTRY_LIMIT: usize = 859;
+const ISSUES_ENTRY_LIMIT: u32 = 1676;
+const ROOT_ENTRY_LIMIT: u32 = 859;
```

Comment:
ideally the `ROOT_ENTRY_LIMIT` should simply not exist since now it's lower than `ENTRY_LIMIT` (as https://github.com/rust-lang/rust/pull/125307 would make it so).

Reply:
Rebased it on top of #125307.